### PR TITLE
Rails react

### PR DESCRIPTION
**Pull request by Kaiserabbas**
# Implement a connection between a Ruby on Rails back-end and React front-end.

- Created a new Rails app called 'hello-rails-react'.
- Used the command `rails new -j webpack my_new_rails_app`.
- Initialized the project with Git.
- Installed needed node dependencies.
- Created a babel.config.js in the root folder of the project.
- Updated webpack.config.js to include the babel loader.
- Created a table for storing the messages and created 10 different greetings.
- Created an API endpoint that selects a random greeting from the table.
- Created a static view that is the root of the app.
#
- Created the App component with react-router and render it in the static view.
- Create the Greeting component that will display a greeting. Set it up as a route in the App component.
- Created a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
- Displayed the random greeting in your Greeting component.

### CODE REVIEW

**Review by mukhammadrizooff — COMMENTED** [review]
Hi @Kaiserabbas, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that your `README` is not descriptive as given by [Microverse Template](https://github.com/microverseinc/readme-template), please implement that one 🙏🏼 

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Kaiserabbas** [comment]
@mukhammadrizooff Thank you!
I have updated the Readme.md as per Microverse template. 
"REVIEW PENDING" on Student Dashboard. I'm unable to submit the new request for the review. 
Kindly review.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
+  "devDependencies": {
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```
Comment (mukhammadrizooff) [inline package.json]: - [ ] [REQUIRED] Good job so far! Kindly you should also add `babel-loader` dependence inside of the `package.json` file as mentioned [this requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=Update%20webpack.config.js%20to%20include%20the%20babel%20loader%20as%20described%20in%20the%20previous%20lesson%27s%20tutorial.) 🙏🏼

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Kaiserabbas, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Kaiserabbas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "react-router-dom": "^6.20.1",
+    "redux": "^5.0.0",
+    "redux-thunk": "^3.1.0",
+    "webpack": "^5.89.0",
+    "webpack-cli": "^5.1.4"
+  },
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
+  "devDependencies": {
```
Comment (Kaiserabbas) [inline package.json:19]: Babel dependecies has been added as per review comment.

**Review by Kaiserabbas — COMMENTED** [review]
Babel dependencies

**Pull request merged**